build-system/global-configs/sw-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cache-service-worker-blacklist": [
```

Comment:
I think this file can be deleted entirely. It's an config file for the removed foreign-fetch service worker.

Reply:
I'll remove it in this PR.